src/components/common/OverallRating/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { ThumbsContainer, RatingThumb } from './Thumb';
+import { OverallRatingContainer, Rating, RatingLabel } from './Rating';
+import styles from './Rating.module.css';
+import PropTypes from 'prop-types';
+
+const thumbs = Array.from({ length: 5 }, (_, i) => i + 1);
+
+const OverallRating = ({ rate, isShowRatingLabel = true }) => {
```

Comment:
default value，可以實作 defaultProps

Reply:
Naming suggestion:
```suggestion
const OverallRating = ({ rate, hasRatingLabel }) => {
```

另外想建議預設 false，要的話就調用
```js
<OverallRating hasRatingLabel />
```
而非反向才 `hasRatingLabel={false}`

Reply:
已實作 defaultProps 並且預設為 false，謝謝建議